charts/hoprd-operator/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: reflector
```

Comment:
I'd advise against this.

Reflector is an unmaintaned beefy image with CVEs. Please, look into the replicator...https://github.com/mittwald/kubernetes-replicator or a better alternative. Also, this is CD level stuff, why is it needed as a dependency?

@sitilge what is your preferred tool for reflecting configs and secrets?

Reply:
Thanks very much, I will use this other one. As I was having a bug in `reflector` where secrets were not sync between namespaces.

Reply:
It's a dependency because I don't want to relay of the end user of the helm chart to know that it should installed it beforehand. It can be disabled with a helm chart variable, in case in the cluster is already installed.